python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/dhcp-servers.j2

```diff
@@ -123,14 +129,21 @@
 {%         include 'eos/dhcp-servers.j2' %}
 ```
 {%     endif %}
-{%     if ethernet_interfaces_dhcp_server | length > 0 %}
+{%     if ethernet_interfaces_dhcp_server | length > 0 or port_channel_interfaces_dhcp_server | length > 0 %}
 
 ### DHCP Server Interfaces
 
 | Interface name | DHCP IPv4 | DHCP IPv6 |
 | -------------- | --------- | --------- |
-{%         for ethernet_interface in ethernet_interfaces_dhcp_server | arista.avd.natural_sort %}
-| {{ ethernet_interface.name }} | {{ ethernet_interface.dhcp_server_ipv4 | arista.avd.default(false) }} | {{ ethernet_interface.dhcp_server_ipv6 | arista.avd.default(false) }} |
-{%         endfor %}
+{%         if ethernet_interfaces_dhcp_server | length > 0 %}
```

Comment:
This `if` is not needed. The for loop would just not loop if the list is empty. Same for v6